SolarCalcs/Calculate_n.py

```diff
@@ -0,0 +1,13 @@
+#Calculate n. The numbered day of the year, for example January 1st is n=1.
+import datetime
+import pytest
+
+def nth_day(year, month, day):
+	mydate=datetime.date(year, month, day)
+	return mydate.strftime('%j')
+
+def test_answer():
```

Comment:
This is a good test. Place in a separate folder/file named tests/Calculate_n.py and import the Calculate_n.py function into that file.